src/redux/store/index.ts

```diff
@@ -9,12 +9,10 @@ import {persistGlobalStore, persistRootReducer} from './persistance';
 
 const configureStore = (initialState?: DeepPartial<WebState>, onReady?: () => void) => {
   const root = persistRootReducer(combineReducers(rootReducer(history)));
-// @ts-ignore
   const composeEnhancers: typeof compose = (typeof window !== 'undefined' && window['__REDUX_DEVTOOLS_EXTENSION_COMPOSE__']) || compose;
   const storeInstance = createStore(
     root,
     initialState,
-    // Code smell on this
```

Comment:
This is completely false, it does change the type expressions and is necessary

Reply:
Yeah, that's why I removed this comment. I was using the comments to help me keep track of what code smells I hadn't fixed yet. Once I realized this was a false positive I removed the comment and marked the smell resolved.